addons/l10n_es_edi_sii/data/template/account.tax-es_common.csv

```diff
@@ -1,94 +1,97 @@
-"account_tax_template_p_iva105_gan","10,5% IVA Soportado régimen ganadero o pesca","","","","","",""
-"account_tax_template_s_iva0_e","IVA 0% Exportaciones","","","exento","","E2","consu"
-"account_tax_template_s_iva0_ic","IVA 0% Entregas Intracomunitarias exentas","","","exento","","E5","consu"
-"account_tax_template_p_req014","1.4% Recargo Equivalencia Compras","","","recargo","","",""
-"account_tax_template_p_req05","0.50% Recargo Equivalencia Compras","","","recargo","","",""
-"account_tax_template_p_req52","5.2% Recargo Equivalencia Compras","","","recargo","","",""
-"account_tax_template_s_irpf1","Retenciones a cuenta IRPF 1%","","","retencion","","",""
-"account_tax_template_s_irpf2","Retenciones a cuenta IRPF 2%","","","retencion","","",""
-"account_tax_template_s_irpf21","Retenciones a cuenta IRPF 21%","","","retencion","","",""
-"account_tax_template_s_irpf21a","Retenciones a cuenta 21% (Arrendamientos)","","","retencion","","",""
-"account_tax_template_s_irpf7","Retenciones a cuenta IRPF 7%","","","retencion","","",""
-"account_tax_template_s_irpf15","Retenciones a cuenta IRPF 15%","","","retencion","","",""
-"account_tax_template_p_irpf1","Retenciones IRPF 1%","","","retencion","","",""
-"account_tax_template_p_irpf15","Retenciones IRPF 15%","","","retencion","","",""
-"account_tax_template_p_irpf21t","Retenciones IRPF (Trabajadores)","","","retencion","","",""
-"account_tax_template_p_iva10_sp_in","IVA 10% Adquisición de servicios intracomunitarios","","","retencion","","",""
-"account_tax_template_p_iva4_sp_in","IVA 4% Adquisición de servicios intracomunitarios","","","retencion","","",""
-"account_tax_template_p_irpf21te","Retenciones IRPF (Trabajadores) en especie","","","retencion","","",""
-"account_tax_template_p_irpf20","Retenciones IRPF 20%","","","retencion","","",""
-"account_tax_template_p_irpf21a","Retenciones 21% (Arrendamientos)","","","retencion","","",""
-"account_tax_template_p_irpf21p","Retenciones IRPF 21%","","","retencion","","",""
-"account_tax_template_p_irpf2","Retenciones IRPF 2%","","","retencion","","",""
-"account_tax_template_s_iva0_isp","IVA 0% Venta con Inversión del Sujeto Pasivo","","","sujeto_isp","","",""
-"account_tax_template_p_iva4_isp","IVA 4% Compra con Inversión del Sujeto Pasivo Nacional","","","sujeto_isp","","",""
-"account_tax_template_p_iva10_isp","IVA 10% Compra con Inversión del Sujeto Pasivo Nacional","","","sujeto_isp","","",""
-"account_tax_template_p_iva21_isp","IVA 21% Compra con Inversión del Sujeto Pasivo Nacional","","","sujeto_isp","","",""
-"account_tax_template_p_rp19","Retenciones 19% (préstamos)","","","retencion","","",""
-"account_tax_template_p_rrD19","Retenciones 19% (reparto de dividendos)","","","retencion","","",""
+"id","name","l10n_es_type","l10n_es_bien_inversion","l10n_es_exempt_reason","tax_scope","active"
+"account_tax_template_s_iva21b","","sujeto",False,"","consu",True

```

Comment:
Here the name is empty, while in later records it is the Spanish instead of the English.  So, better to remove it entirely.

Reply:
When I remove the name column I get a database error saying name is mandatory. But I could leave them all empty